Description: fix an armhf build failure, by reverting the pld function removal.
../../client/vector/analyzeFuncs_vfp_aux.cpp: In function 'void v_pfsubTranspose(float*, float*, int, int)':
../../client/vector/analyzeFuncs_vfp_aux.cpp:68:24: error: there are no arguments to 'pld' that depend on a template parameter, so a declaration of 'pld' must be available [-fpermissive]

--- boinc-app-seti-8.00~svn3363.orig/client/vector/analyzeFuncs_vfp_aux.cpp
+++ boinc-app-seti-8.00~svn3363/client/vector/analyzeFuncs_vfp_aux.cpp
@@ -48,6 +48,14 @@
 #include "asmlib.h"
 #include "pulsefind.h"
 
+inline void pld(void *arg1,const int arg2=0) {
+    __asm__ __volatile__ (
+      "pld [%0,%1]\n"
+      : 
+      : "r" (arg1), "Jr" (arg2)
+    );
+}
+
 
 template <int x>
 inline void v_pfsubTranspose(float *in, float *out, int xline, int yline) {
